.travis.yml

```
@@ -8,7 +8,7 @@ script:
 - bundle exec rspec
 deploy:
   provider: heroku
-  api_key:
+  api_key: 5c0161f1db69f834bfa0c6a5ceee7187
```

```suggestion
  api_key: 
```
This should be able to work with the ```secure:``` right below it. We don't want to upload our API key to Github. If it doesn't work I can help you fix it but this shouldn't be necessary.